<launch>

    <!-- Launch GPS Manager -->
    <node name="gps_manager" pkg="decision" type="gps_manager" output="screen">
        <rosparam param="at_goal_tolerance">0.5</rosparam>
        <rosparam param="waypoints">
        [-123.248523249, 49.2627501538, -123.248591436, 49.2625626029]
        </rosparam>
    </node>

</launch>
